deployment/terraform/ssh/ssh.go

```diff
@@ -71,43 +71,51 @@ func (ea *ExtAgent) NewClient(ip string) (*Client, error) {
 }
 
 // RunCommand runs a given command in a new ssh session.
-func (sshc *Client) RunCommand(cmd string) error {
+func (sshc *Client) RunCommand(cmd string) (string, error) {
 	sess, err := sshc.client.NewSession()
 	if err != nil {
-		return err
+		return "", err
 	}
 	defer sess.Close()
+	var out strings.Builder
+	sess.Stdout = &out
+	sess.Stderr = &out
 
-	return sess.Run(cmd)
+	err = sess.Run(cmd)
+	return out.String(), err
 }
 
 // Upload uploads a given src object to a given destination file.
-func (sshc *Client) Upload(src io.Reader, dst string, sudo bool) error {
+func (sshc *Client) Upload(src io.Reader, dst string, sudo bool) (string, error) {
```

Comment:
Nit: To be more consistent with similar methods like `cmd.Output()`, maybe we can return `[]byte`.

Reply:
Sure.